recipes/qxmpp/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(qxmpp)
+
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Does qxmpp require C++ 17? If yes, you need to validate it first on the recipe. Take it as example: https://github.com/conan-io/conan-center-index/blob/41d4fbb8c218f9d938ccb8b9d888bff52002e62e/recipes/cppitertools/all/conanfile.py#L30

Reply:
it does, because qt6 requires c++17